examples/axon/README.md

```diff
@@ -69,12 +69,11 @@ entries.
 
 We provide a small demo to let you test the program. It consists of the
 following scripts:
-- [run_demo.sh] (run_demo.sh): compiles the P4 program, starts the switch,
-  configures the data plane by running the CLI [commands]
-  (commands.txt), and starts the mininet console.
-- [receive.py] (receive.py): listens for Axon formatted packets.  This
+- [run_demo.sh](run_demo.sh): compiles the P4 program, starts the switch,
+  configures the data plane by running the CLI [commands](commands.txt), and starts the mininet console.
```

Comment:
I didn't consider about the restriction of the line characters. I moved some blank spaces but I think there may be some interruptions when testing. So this PR is not very suitable and I will close it.

Thank you, Antonin!